llvm/lib/ObjCopy/DXContainer/DXContainerObjcopy.cpp

```diff
@@ -28,6 +52,31 @@ static Error handleArgs(const CommonConfig &Config, Object &Obj) {
       return Config.ToRemove.matches(P.Name);
     };
 
+  if (!Config.SplitSection.empty()) {
+    for (StringRef Flag : Config.SplitSection) {
+      StringRef SectionName;
+      StringRef FileName;
+      std::tie(SectionName, FileName) = Flag.split('=');
+
+      if (Error E = splitPartAsObject(SectionName, FileName,
+                                      Config.InputFilename, Obj))
+        return E;
+    }
+
+    RemovePred = [&Config, RemovePred](const Part &P) {
```

Comment:
Reusing the same `RemovePred` here is a bit confusing.
```suggestion
    RemovePred = [&Config, OriginalRemovePred](const Part &P) {
```

Reply:
This follows the implementation similar to [ELFObjcopy](https://github.com/inbelic/llvm-project/blob/bfd490e0cda0917dfc357ca87f3c0639ba38a56f/llvm/lib/ObjCopy/ELF/ELFObjcopy.cpp#L414), in preparation for when it may be extended in the future.